sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/TypeFilter.java

```diff
@@ -0,0 +1,39 @@
+package com.mercedesbenz.sechub.commons.model;
+
+public interface TypeFilter {
+    public static class AcceptAllTypeFilter implements TypeFilter {
+
+        @Override
+        public boolean isTypeAccepted(String type) {
+            return true;
+        }
+    }
+
+    public static class AcceptTypeFilter implements TypeFilter {
+
+        private String type;
+
+        public AcceptTypeFilter(String type) {
+            this.type = type;
+        }
+
+        @Override
+        public boolean isTypeAccepted(String type) {
+            if (type == null) {
+                return false;
+            }
+            // more than one type can be defined eg. docker,git,binary
+            final String[] splitTypes = type.split(",");
+            for (String splitType : splitTypes) {
```

Comment:
You do not handle situations where a setup contains whitespaces inside, e.g. "docker, git, binary"

Please add
- trim mechanism
- unit test which has a comma separated list inside which contains whitespaces
```suggestion
            for (String splitType : splitTypes) {
                 String typeToCheck = splitType.trim();
```